DXRando/DeusEx/Classes/Player.uc

```diff
@@ -248,6 +262,17 @@ function bool DXReduceDamage(int Damage, name damageType, vector hitLocation, ou
 
         if (augLevel >= 0.0)
             newDamage *= augLevel;
+
+        if (UsingChargedPickup(class'HazMatSuit'))
+        {
+            skillLevel = SkillSystem.GetSkillLevelValue(class'SkillEnviro');
+            newDamage *= 0.75 * skillLevel;
+        }
+        else // passive enviro skill still gives some damage reduction
+        {
+            skillLevel = SkillSystem.GetSkillLevelValue(class'SkillEnviro');
+            newDamage *= (skillLevel + 1)/2;
+        }
```

Comment:
here inside
```
    if ((damageType == 'Burned') || (damageType == 'Flamed') ||
        (damageType == 'Exploded') || (damageType == 'Shocked'))
```
I'm copying the code from above so that the hazmat suit and the enviro skill here with these damage types too, and this does stack with AugShield, but AugEnviro does not help with these damage types at all